blockchain/blockchain.go

```diff
@@ -278,7 +278,7 @@ func (bc *blockchain) TipHash() hash.Hash256 {
 func (bc *blockchain) TipHeight() uint64 {
 	tipHeight, err := bc.dao.Height()
 	if err != nil {
-		log.L().Panic("failed to get tip height", zap.Error(err))
+		log.L().Error("failed to get tip height", zap.Error(err))
```

Comment:
what can we do instead of panic if we can not get tip height?

Reply:
changed it back

there is a wrong e2etest, it is root-caused and fixed